### PR TITLE
 Add default seconds to liveness and readiness probes

### LINKED ISSUE #1092: Add failureThreshold and timeoutSeconds  to liveness and readiness probe configs in values yaml
### Problem

The current chart does not set the failureThreshold and timeoutSeconds values in the liveness and readiness probe configs in values yaml

### Solution

Add failureThreshold and timeoutSeconds in the liveness and readiness probe configs in values yaml

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Nana-EC**
**Description**:
 Currently liveness and readiness probes where missing default seconds

To the liveness and readiness probes
- Add initialDelaySeconds
- Add periodSeconds
- Add timeoutSeconds
- Add failureThreshold

**Related issue(s)**:

Fixes #1092 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: helm-chart/values.yaml [hunk 1]
```diff
@@ -137,3 +137,20 @@ test:
   enabled: false
   schedule: '@daily'
   baseUrl: "http://127.0.0.1:7546"
+
+livenessProbe:
+  httpGet:
+    path: /health/liveness
+    port: jsonrpcrelay
+  initialDelaySeconds: 50
```
Comment (steven-sheehy) [inline helm-chart/values.yaml:145]: The initialDelaySeconds seems high. Does your app need that time to initialize? Maybe closer to 20 for each?

File: helm-chart/values.yaml [hunk 2]
```diff
@@ -137,3 +137,20 @@ test:
   enabled: false
   schedule: '@daily'
   baseUrl: "http://127.0.0.1:7546"
+
+livenessProbe:
```
Comment (steven-sheehy) [inline helm-chart/values.yaml:141]: nit: Prefer them sort inline like the rest of the file instead of at the end

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: helm-chart/values.yaml (hunk 2 above)
Comment (Nana-EC) [reply helm-chart/values.yaml:141]: Done

Comment (steven-sheehy) [reply helm-chart/values.yaml:141]: I meant the top level key within the entire file, but that's fine 😄

**Review by steven-sheehy — APPROVED** [review]
LGTM

File: helm-chart/values.yaml (hunk 2 above)
Comment (Nana-EC) [reply helm-chart/values.yaml:141]: Next update to the chart

Comment (ebadiere) [reply helm-chart/values.yaml:141]: LGTM

**Review by lukelee-sl — APPROVED** [review]

**Pull request merged**